game/Assets/Scripts/HomeScreenButtonManager.cs

```diff
@@ -33,6 +33,6 @@ private void StartGame()
     // Callback for the learn button
     private void Learn()
     {
-        throw new System.NotImplementedException();
+        UnityEngine.SceneManagement.SceneManager.LoadScene("LearnScene");
```

Comment:
We can probably merge `HomeScreenButtonManager` and `ButtonManager` to one script which handles the button logic (topics, start game and learn).

I created this script initally since I wanted to keep it seperate from `TopicManager` but I see you renamed it in this PR.